doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,96 @@
+"""
+============================================================
+Reconstruct dust-covered human cornea image using inpainting
+============================================================
+
+Optical Coherence Tomography (OCT) [1]_ is used to provide eye doctors with an image
+of the retina in the back of a patient's eye. It utilizes the concept of inferometry
+to create a cross-sectional map of the retina that is accurate to within at least 10-15
+microns. From its inception, OCT images have been acquired in a time domain fashion. It is
+useful in the diagnosis of many retinal conditions, especially when the media is clear.
+
+Quite commonly, dust gets accumulated on the reference mirror of the equipment and causes
+dark spots to appear on images. This could reduce the accuracy of an optometrist's diagnosis.
+In this example, we reproduce the steps taken to perform OCT dust removal in an image to
+restore it to its original form. This application was first discussed by Jules Scholler
+in [2]_.
+
+.. [1] Vinay A. Shah M.D. (2015)
+       `Optical Coherence Tomography <https://eyewiki.aao.org/Optical_Coherence_Tomography#:~:text=3%20Limitations-,Overview,at%20least%2010%2D15%20microns.>`_,
+       American Academy of Ophthalmology.
+.. [2] Jules Scholler (2019) "Image denoising using inpainting":
+       `<https://www.jscholler.com/2019-02-28-remove-dots/>`_
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import imageio.v3 as iio
+import plotly.express as px
+
+
+#################################################################################
+# Let’s define a convenience function for plotting comparisons:
+
+
+def plot_comparison(original, filtered, title1, title2):
+
+    fig, (ax1, ax2) = plt.subplots(ncols=2, figsize=(12, 8), sharex=True, sharey=True)
+    ax1.imshow(original)
+    ax1.set_title(title1)
+    ax1.axis('off')
+    ax2.imshow(filtered)
+    ax2.set_title(title2)
+    ax2.axis('off')
+
+
+#################################################################################
+
+# Load image
+# ==========
+# The dataset that we are using in this example is an image sequence showing the palisades of
+# Vogt in a human cornea in vivo. The file has been acquired in TIFF format.
```

Comment:
@ana42742 @decorouz I believe that an older comment of mine (in the HackMD draft) wasn't really on point, because:

* it's the *data* that are *acquired*;
* the data are saved into a TIFF file;
* I'm not sure this information is worth adding (we can see from the file extension that it's TIFF, and we don't have details about the acquisition technique).